sdk/core/azure-core/CMakeLists.txt

```diff
@@ -64,17 +73,25 @@ add_library (
   src/version.cpp
   )
 
-target_include_directories (${TARGET_NAME} PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/inc> $<INSTALL_INTERFACE:include/az_core>)
+target_include_directories(
+  ${TARGET_NAME}
+    PUBLIC
+      $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/inc>
+      $<INSTALL_INTERFACE:include>
+  )
 
 # make sure that users can consume the project as a library.
-add_library (Azure::Core ALIAS ${TARGET_NAME})
+add_library(Azure::${TARGET_NAME} ALIAS ${TARGET_NAME})
```

Comment:
I don't think this is OK.  the Alias should be Azure::Core and not Azure::azure-core